source/docs/reference/configurations.md

```diff
@@ -0,0 +1,107 @@
+---
+id: configurations
+title: Configurations
+---
+
+<blockquote class="note">
+This documentation is the current state of the configurations available in Front-Commerce. However, our goal is to clean it up to make sure that it is still easily customizable, but also easier to check if your configurations are correct. We will provide a clear migration path and will integrate this change within our 1.0.0 release.
+
+Please refer to the [related gitlab issue](https://gitlab.com/front-commerce/front-commerce/issues/106) to follow its advancement.
+</blockquote>
+
+## How to set your configurations
```

Comment:
I think we should a bold warning here, either in introduction or in that most of these configurations are included in the client bundle. Thus it is important to not include private configurations in them, and to use env var instead.

It could also be a good reminder to add in the `website.js` section.